app/schema.graphql

```diff
@@ -1450,6 +1450,9 @@ type User implements Node {
   email: String!
   username: String
   createdAt: DateTime!
+
+  """API keys associated with the user."""
+  apiKeys: [ApiKey!]
```

Comment:
```suggestion
  apiKeys: [ApiKey!]!
```
empty lists are the same as nullable. It's much simpler to deal with

Reply:
I originally had it as a resolver but wouldn't that mean when we covert each user orm object to a gql node it'll run a synchronous query to fetch the api keys?